test/registration/hosts_api_test.rb

```diff
@@ -0,0 +1,29 @@
+require 'test_helper'
+require 'registration/registration_hosts_api'
+
+class RegistrationHostsApiTest < Test::Unit::TestCase
+  include Rack::Test::Methods
+
+  def app
+    Proxy::Registration::HostsApi.new
+  end
+
+  def setup
+    @foreman_url = 'http://foreman.example.com'
+    Proxy::SETTINGS.stubs(:foreman_url).returns(@foreman_url)
+  end
+
+  def test_get_hosts_api
+    stub_request(:get, "#{@foreman_url}/api/hosts").to_return(:body => 'GET hosts')
```

Comment:
Can we verify the body is returned in `last_response`? A pretty crucial part of this.